internal/datastore/common/sql.go

```diff
@@ -64,86 +64,159 @@ const (
 
 // SchemaInformation holds the schema information from the SQL datastore implementation.
 type SchemaInformation struct {
-	colNamespace         string
-	colObjectID          string
-	colRelation          string
-	colUsersetNamespace  string
-	colUsersetObjectID   string
-	colUsersetRelation   string
-	colCaveatName        string
-	paginationFilterType PaginationFilterType
+	RelationshipTableName string
+	ColNamespace          string
+	ColObjectID           string
+	ColRelation           string
+	ColUsersetNamespace   string
+	ColUsersetObjectID    string
+	ColUsersetRelation    string
+	ColCaveatName         string
+	ColCaveatContext      string
+	PaginationFilterType  PaginationFilterType
+	PlaceholderFolder     sq.PlaceholderFormat
```

Comment:
Folder?

Reply:
Also is this in here because this allows you to push more logic up into the common sql logic?

Reply:
Yes